{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2024-47398",
      "ASSIGNER": "scy@openharmony.io"
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-787"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://gitee.com/openharmony/security/blob/master/zh/security-disclosure/2025/2025-01.md",
          "name": "https://gitee.com/openharmony/security/blob/master/zh/security-disclosure/2025/2025-01.md",
          "refsource": "",
          "tags": []
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "in OpenHarmony v4.1.2 and prior versions allow a local attacker cause the device is unable to boot up through out-of-bounds write."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "impact": {},
  "publishedDate": "2025-01-07T08:15Z",
  "lastModifiedDate": "2025-01-07T08:15Z"
}